### PR TITLE
Peanuts and peanut butter! + Other various new foods

### PR DESCRIPTION
**Pull request by MidoriWroth**
## About The Pull Request

This PR will:

- Add growable peanuts! They can be grown in botany and eaten as is, or ground into peanut butter to be used for various other recipes. Peanut butter can also be ordered at the order console for 30 credits per jar.
- Add a few new peanut related recipes, including: Peanut butter and jelly sandwich, peanut butter and banana sandwich, peanut butter cookies, and granola bars.
- Add brownies! A treat that has been long neglected from the game. Initially the batter is made, which must be baked in an oven and then sliced into pieces.
- Add breadsticks! A simple side to make for many dishes.
- Add nigiri sushi, for a little more sushi variety as per many requests.
- Changed the meatball noodle soup to require peanuts instead of Gallery's Peanuts

image

## Why It's Good For The Game

More foods add more variety to the possible menus chefs can make. I was surprised that peanuts and brownies were not a thing in the game yet, so I had to gadd them

## Changelog

:cl:
add: Added growable peanuts! Grow them in botany for a snack, or grind them into peanut butter
add: Added 4 different peanut and peanut butter-based recipes
add: Added brownies. Yes, I know, finally
add: Added breadsticks, as if food addiction couldn't get worse
add: Added a simple nigiri sushi, for a little more sushi variety. Expect more in the future.
qol: Meatball noodle soup now requires grown peanuts instead of Gallery's peanuts
imageadd: New sprites for all the new foods!
/:cl:

### CODE REVIEW

**Comment by newfren** [comment]
there's a bag of peanuts in the junk food vendors that's used in a couple recipes

**Comment by Wallemations** [comment]
Now all I need is celery and we can make literal ants on a log

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 1]
```diff
@@ -1063,3 +1063,10 @@
 	color = "#adcf77"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
+/datum/reagent/consumable/peanut_butter
```
Comment (EOBGames) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm:1066]: peanut butter is already a reagent, added in #59412 - it's a few above this new define in the code, specifically between whipped cream and vinegar, so adding this is unnecessary

File: code/game/objects/items/food/bread.dm [hunk 2]
```diff
@@ -313,8 +313,32 @@
...
 	icon = 'icons/obj/food/burgerbread.dmi'
 	icon_state = "frenchtoast"
-	food_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/nutriment/vitamin = 2,)
+	food_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/nutriment/vitamin = 2)
 	tastes = list("french toast" = 1, "syrup" = 1, "golden deliciousness" = 1)
 	foodtypes = GRAIN | BREAKFAST
 	w_class = WEIGHT_CLASS_SMALL
 	burns_on_grill = TRUE
+
+/obj/item/food/rawbreadstick
```
Comment (GoldenAlpharex) [inline code/game/objects/items/food/bread.dm:322]: Always use `snake_case` whenever you can, when writing typepaths and variable names.

There's a lot of them so I won't comment on each of them, but if you need help figuring it out, hit me up.
```suggestion
/obj/item/food/raw_breadstick
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by MidoriWroth** [comment]
yeah idk how i didn't realize that peanut butter was already a thing lmao, played this game for years and never once saw it.

**Comment by Sucy-Manbavaran** [comment]
Looking at the code, im a bit inspired on adding some new foods! i didn't realize how simple it could be ahaha, though i see a lot of places where i could make mistakes! And i do hope this gets merged! these are some very fun sounding recipes and i'd love to be able to make them!

**Comment by san7890** [comment]
image

The bacon updates aren't mentioned in your PR (via IconDiffBot2), is this intentional? It also appears you might be reverting #68255 in regards to the mint.

**Comment by MidoriWroth** [comment]
> image
> 
> The bacon updates aren't mentioned in your PR (via [IconDiffBot2](https://github.com/tgstation/tgstation/pull/68289/checks?check_run_id=7266185312)), is this intentional? It also appears you might be reverting #68255 in regards to the mint.

oh! uhhh, no, that was not intentional. let me see if i can fix it

**Comment by MidoriWroth** [comment]
> Looking at the code, im a bit inspired on adding some new foods! i didn't realize how simple it could be ahaha, though i see a lot of places where i could make mistakes! And i do hope this gets merged! these are some very fun sounding recipes and i'd love to be able to make them!

yeah it's very simple! i'd like to see more stuff added, so go for it!

**Comment by MidoriWroth** [comment]
image

New peanut butter jar sprite courtesy of @ArcaneMusic who is the GOAT

**Comment by tf-4** [comment]
That peanut butter jar seems a bit uh, huge, as good as the sprite is

**Comment by MidoriWroth** [comment]
i'll see if i can make it a bit smaller

**Comment by MidoriWroth** [comment]
so i made this one, quite pleased with it, it's somewhat of a smaller sprite 
image

**Comment by MidoriWroth** [comment]
I fucked this up real bad lmao, just gonna start over

**Comment by Sucy-Manbavaran** [comment]
> I fucked this up real bad lmao, just gonna start over

godspeed, i'll keep in mind NOT to base any food related coding i do off your code 😆

**Pull request closed without merging**